test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,16 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+        isAnyNull("abc", 555, null);
```

Comment:
Test cases for isAnyNull() method from Utils.java, should be in separate method in UtilsTest.java. You may follow this practice that for every method being tested there should be a corresponding method of the same name in the test.java file. The problem with your current version is that you do not separate the method which tests for elementsAreUnique() and isAnyNull(), which is not a good practice. You should be more careful about separation of concerns for each test method.